src/clr-icons/shapes/core.ts

```diff
@@ -342,6 +342,20 @@ export const ClrShapeEye = { eye: ClrShapeEyeSVG };
 export const ClrShapeEyeHide = { 'eye-hide': ClrShapeEyeHideSVG };
 export const ClrShapeStepForward2 = { 'step-forward-2': ClrShapeStepForward2SVG };
 
+Object.defineProperty(ClrShapeHome, 'house', descriptorConfig(ClrShapeHomeSVG));
```

Comment:
Thank you, Jeeyun! :D Have we tested it by importing an individual shape? For example, if we import an individual shape, we expect both its icon name and aliases are added? 

I am a bit concerned that had the object been already exported, the alias would be actually missing because we are assigning the alias after the export.
```
export const ClrShapeHome = { home: ClrShapeHomeSVG };
Object.defineProperty(ClrShapeHome, 'house', descriptorConfig(ClrShapeHomeSVG));
```

Reply:
The old way worked too where aliases were added after the setShapes was exported. This is because the Object.defineProperty method mutates via the reference so it trickles to the exported value as well.

Reply:
You might be correct there. But also I think that could have been case due to this block of code. We had this in every set of icons file. We did all our alias assignments before this:
```ts
if (typeof window !== 'undefined' && window.hasOwnProperty('ClarityIcons')) {
    window.ClarityIcons.add(ChartShapes);
}
```
Now icons are separate and individually inserted so I think it's worth testing icon and its alias by importing by their individual host object.

Reply:
Yesss! Awesome 👍